client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/components/credentials-form.tsx

```diff
@@ -45,6 +52,8 @@ export const CredentialsForm: FC< CredentialsFormProps > = ( { onSubmit, onSkip
 		return translate( 'Continue' );
 	};
 
+	const showSpecialInstructions = ! applicationPasswordEnabled || accessMethod === 'backup';
+
 	return (
 		<form className="site-migration-credentials__form" onSubmit={ submitHandler }>
```

Comment:
What do you think about having separate forms for processing the different behaviors?

Reply:
We had a call in which we decided to have the form refactor not part of this project.